doc/source/io.rst

```diff
@@ -19,6 +19,12 @@ Any arguments passed to ``read_file()`` after the file name will be passed direc
 
 Among other things, one can explicitly set the driver (shapefile, GeoJSON) with the ``driver`` keyword, or pick a single layer from a multi-layered file with the ``layer`` keyword.
 
+Where supported in ``fiona`` *geopandas* can also load resources directly from
```

Comment:
I would put a comma between fiona and geopandas

Reply:
Done.